src/scripts/Display/cards.js

```diff
@@ -0,0 +1,53 @@
+import tv from '../API/TV-maze.js';
+import involvement from '../API/Involvement.js';
+import elisteners from '../utils/listeners.js';
+import counter from '../utils/counter.js';
+
+class CardsUX {
+  constructor() {
+    this.itemsList = document.querySelector('.items-list');
+  }
+
+  renderCards = async () => {
+    const arrayOfShows = await tv.getAllShows();
+    const arrOfMoviesToDsiplay = arrayOfShows.slice(0, 49);
+    const arrayOfLikes = await involvement.getLikes();
+    const numberOfMovies = await counter.moviesCounter();
+    const numberOfMoviesInDisplay = counter.countElements(arrOfMoviesToDsiplay);
+    document.querySelector('.alert').innerHTML = `You are seeing ${numberOfMoviesInDisplay} of the ${numberOfMovies} movies we have for you`;
+    arrOfMoviesToDsiplay.forEach((show) => {
+      let numOfLikes = 0;
+      if (arrayOfLikes.some((element) => element.item_id === show.id)) {
+        numOfLikes = arrayOfLikes.find((like) => like.item_id === show.id).likes;
+      }
+
+      const clone = this.itemsList.firstElementChild.cloneNode(true);
+      clone.classList.remove('d-none');
+      const footer = clone.querySelector('.card-footer');
+      const commentBtn = footer.querySelector('a');
+      commentBtn.addEventListener('click', () => {
+        const inputForm = document.querySelector('.form');
+        const viewCommentsBtn = document.querySelector('.display-comment-btn');
+        inputForm.dataset.id = show.id;
+        viewCommentsBtn.dataset.id = show.id;
```

Comment:
**Remember to reflect your changes in the live demo too. For both points, I have added a Loom Video link for better elaboration (below):**

- [ ] I think your comments modal implementation has a bug. When I comment on a movie and retrieve the comments, and go to another movie modal I see comments from the previous movie, but that should be the case, as stated in requirements (see image). Kindly fix that. It is merely my suggestion that you call your `displayCommentCount()` and `displayAllComments()` functions upon modal load, depending on how your code is implemented.

  ![ricky-comm](https://user-images.githubusercontent.com/36542986/148204748-8d8fb9cc-f1b0-4ba5-9862-0b5b5595d42c.jpg)

- [ ] Also, after navigating from a movie modal that has comments to one that does not I still see the comments count from the previous movie modal even after clicking **`View comments`** button. Kindly fix that.

**Loom Video Link: [here](https://loom.com/share/1027ae26b6474d129777b9fdb0729e4c)**